### ISSUE TITLE
could not found requirements.txt anywhere

### ISSUE THREAD

**Issue by rajni-keyss** [imartinez/privateGPT#1423]
I am trying to install dependencies but could not found requirements.txt anywhere

**Comment by bhadane-vivek** [comment]
Hey @rajni-keyss!
This project uses poetry so if you look at the `pyproject.toml` file it shows all the dependencies.

**Comment by KorruptedPineapple** [comment]
Hi @rajni-keyss, fun fact, this is my first github comment ever. But I believe you're following Matthew Bermans old video on youtube? The current privateGPT version is far newer than that old video.

**Comment by rajni-keyss** [comment]
@KorruptedPineapple yes you are right, but now i get the installation guide and i have installed it.. thanks .. this is the path /fern/docs/pages/installation/installation.mdx
